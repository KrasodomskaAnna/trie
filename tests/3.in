+ cverjf hxl
+ xgnqx jjn
+ cvmyp lhu
+ reugf pi
+ xgsgkx layo
+ cvzyu vmbi
+ rel jlq
+ cvsv tjkf
+ xgwh k
+ rea nh
+ xgoj okzr
+ cvoal b
+ cvlbuy qp
+ rexj tv
+ cvij sl
+ rerh rfes
+ xgdvcu wks
+ xgwq s
+ xgm pqgf
+ cvszkk n
+ xgdfew assq
+ xgheq fhz
+ cvyjra ih
+ cvvtz al
+ xgzwpr y
+ retwd wsof
+ cvp r
+ cvewz v
+ xgasd ea
+ xgw du
? pcme
? keis
? mq
? rea
? cvp
? xgm
? owso
? tftx
? d
? cvp
? rel
? cvp
? xgw
? xgm
? cvp
? rel
? n
? rea
? myxc
? euls
? cvp
? rea
? z
? haz
? rel
? vm
? e
? xrx
? rel
? ldf
? xgm
? xgw
? rgw
? cvp
? cvp
? gpbm
? rea
? rea
? cg
? rea
? xgw
? rel
? xgw
? xgw
? cvp
? cvp
? cvp
? qrhd
? cvp
? cvp
? wy
? h
? rel
? m
? pqjb
? cvp
? rea
? xgm
? xgw
? xgm
* rerh
* a
* t
* z
* xgw
* xg
* cvp
* yf
* c
* cvp
