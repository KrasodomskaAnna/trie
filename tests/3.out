-
-
-
nh
r
pqgf
-
-
-
r
jlq
r
du
pqgf
r
jlq
-
nh
-
-
r
nh
-
-
jlq
-
-
-
jlq
-
pqgf
du
-
r
r
-
nh
nh
-
nh
du
jlq
du
du
r
r
r
-
r
r
-
-
jlq
-
-
r
nh
pqgf
du
pqgf
rfes
-
-
-
du
k
s
ea
assq
wks
fhz
pqgf
jjn
okzr
layo
du
k
s
y
r
-
hxl
v
sl
qp
lhu
b
r
tjkf
n
al
ih
vmbi
r